cmd/clusterctl/client/config/providers_client.go

```diff
@@ -72,6 +72,7 @@ const (
 	K0smotronProviderName      = "k0sproject-k0smotron"
 	IonosCloudProviderName     = "ionoscloud-ionoscloud"
 	VultrProviderName          = "vultr-vultr"
+	OpenNebulaProviderName     = "opennebula"
```

Comment:
It should contain the github org to distinct from potential community projects:

https://main.cluster-api.sigs.k8s.io/developer/providers/contracts/clusterctl.html?highlight=provider#providers-github-org-prefix
```suggestion
	OpenNebulaProviderName     = "opennebula-opennebula"
```